lib/resources/example_json/r4_examples_relatedperson.rb

```diff
@@ -538,6 +580,22 @@ module Resources
               'code': '1152',
               'display': 'Emergency Contact'
             }
+          ],
+          'extension': [
+            {
+              'valueCodeableConcept': {
+                'coding': [
+                  {
+                    'system': 'https://fhir.cerner.com/2c400054-42d8-4e74-87b7-80b5bd5fde9f/codeSet/40',
+                    'cod': '156',
+                    'display': 'Mother',
+                    'userSelected': true
+                  }
+                ],
+                'text': 'Mother'
```

Comment:
Let's not put display, userSelected, and text since I don't believe we actually write those values.

Reply:
removed here https://github.com/cerner/fhir.cerner.com/pull/644/commits/767fd43d02089bf605dfdde72918fc393bec5655#diff-133ac83b73a8a88b430a777cf1700b9010258bd96bc4ff69a51350091f73643eL85-L86 https://github.com/cerner/fhir.cerner.com/pull/644/commits/767fd43d02089bf605dfdde72918fc393bec5655#diff-133ac83b73a8a88b430a777cf1700b9010258bd96bc4ff69a51350091f73643eL89

Reply:
Looks like display and userSelected are still here: https://github.com/cerner/fhir.cerner.com/commit/767fd43d02089bf605dfdde72918fc393bec5655#diff-133ac83b73a8a88b430a777cf1700b9010258bd96bc4ff69a51350091f73643eR1035-R1036

Reply:
I also don't believe we support standard codes on creates

Reply:
> Looks like display and userSelected are still here: [767fd43#diff-133ac83b73a8a88b430a777cf1700b9010258bd96bc4ff69a51350091f73643eR1035-R1036](https://github.com/cerner/fhir.cerner.com/commit/767fd43d02089bf605dfdde72918fc393bec5655#diff-133ac83b73a8a88b430a777cf1700b9010258bd96bc4ff69a51350091f73643eR1035-R1036)

removed it in later commits https://github.com/cerner/fhir.cerner.com/commit/8826890486713f37b02cddcff2f7f3870f42cb16#diff-133ac83b73a8a88b430a777cf1700b9010258bd96bc4ff69a51350091f73643eL1041-L1042

Reply:
> I also don't believe we support standard codes on creates

removed it here https://github.com/cerner/fhir.cerner.com/pull/644/commits/8826890486713f37b02cddcff2f7f3870f42cb16#diff-133ac83b73a8a88b430a777cf1700b9010258bd96bc4ff69a51350091f73643eL1036-L1042

Reply:
+1.